scales-other.qmd

```diff
@@ -264,14 +266,16 @@ linetypes <- function(n) {
   return(types[seq_len(n)])
 }
 
-base + scale_linetype(palette = linetypes)
+base + discrete_scale("linetype", palette = linetypes)
```

Comment:
Per https://github.com/tidyverse/ggplot2/issues/5991#issuecomment-2220305051, this usage was incorrect but behaved correctly by accident. There currently isn't a linetype scale template that allows setting palettes directly, as is the case for many aesthetics IIRC. We can enable this, but it'd have to wait for the next release.